crates/services/txpool/src/lib.rs

```diff
@@ -29,7 +30,10 @@ pub mod mock_db;
 pub use mock_db::MockDb;
 
 pub use config::Config;
-pub use fuel_core_types::services::txpool::Error;
+pub use error::{
+    Error,
```

Comment:
```suggestion
    Error as TxPoolError,
```

Reply:
I prefer not to type alias the error here, since we're just re-exporting the error type from the root of the `txpool` crate. Instead, I would rather let type aliasing happen in modules importing the error and result type.